app/assets/javascripts/application.js

```diff
@@ -14,3 +14,106 @@
 //= require jquery_ujs
 
 //= require_tree .
+$(document).ready(function() {
+
+  var x = document.getElementById("demo");
+  var minUpdateUserDistance = 0.01 //km
+  var userId = $("div[style='display:none']").attr('id')
+  var firebaseRef = new Firebase("https://blinding-fire-43.firebaseio.com/");
+  var geo_options = {
+  enableHighAccuracy: true,
```

Comment:
Also, be consistent with naming.  Most of your JS objects are using camel case (`firebaseRef`), but here you are using snakecase (`geo_options`), below you are just scrunching words together (`currentdate`) :smile:   I don't care which you choose, but choose one and be consistent.  This is where code styleguides help.  Also, when you review each other's code, you should check and be aware of that.